### PR TITLE
Prepare for v0.2.0 release

### PR DESCRIPTION
**Pull request by pratikunterwegs**
This PR contains edits preparing the repo for a v0.2.0 GH release.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -36,7 +53,15 @@ No substantial changes to classes; small additions of input checking to `<popula
...
-3. All function documentation has been updated to reflect name changes and other minor improvements.
+3. A design decisions vignette has been added to help developers and contributors understand the package architecture and design choices made in _epidemics_ development; this includes a conceptual design diagram as well as an architecture diagram.
+
+4. All function documentation has been updated to reflect name changes and other minor improvements.
+
+5. The README has been reorganised to shift the model list to the end, to make the installation instructions easier to find.
+
+6. Corrected the website URL in `_pkgdown.yml`; this allows search functionality in the package website.
+
+7. Updated WORDLIST.
```
Comment (joshwlambert) [inline NEWS.md:64]: Do these have PRs that could be linked?

File: NEWS.md [hunk 2]
```diff
@@ -46,7 +71,11 @@ No substantial changes to classes; small additions of input checking to `<popula
 
 3. Removed {deSolve} from dependencies.
 
-4. Updated NEWS.md file to track changes to the package.
+4. Added {ggdist}, {withr}, {purr} and {tidyr} to Suggests; `CITATION.cff` updated to match.
+
+5. Added basic infrastructure for continuous relative benchmarking, but this is a work in progress (see issue #184).
+
+6. Updated NEWS.md file to track changes to the package.
```
Comment (joshwlambert) [inline NEWS.md:78]: This can be removed, the updates to `NEWS.md` are not normally reported in the `NEWS.md`.

**Review by joshwlambert — APPROVED** [review]
I haven't tracked the developments in {epidemics} from v0.1.0 to v0.2.0 so cannot check whether all developments are covered in the `NEW.md` points, but it reads clearly and contains a good amount of detail, all looks good to be merged.

I've left a few minor comments in the files.

File: NEWS.md [hunk 3]
```diff
@@ -1,20 +1,37 @@
...
 
 4. The function `get_parameter()` has been removed.
 
+5. The infection parameter "transmissibility" has been renamed to "transmission rate" and the corresponding function argument is `transmission_rate`; see PR #196.
+
+## Model functions
+
+- ODE model functions can now be passed numeric vectors of infection parameters, and lists of intervention sets (a list of `<intervention>`s), and lists of `<vaccination>` to the `intervention` and `vaccination` argument respectively. This is the **main change in this minor version.**
+
+- ODE model functions take on {data.table} to make combinations of interventions and vaccinations (together called a 'scenario'), and infection parameter sets to run each scenario for each parameter set.
```
[the remark this answers is not in the document]
Comment (pratikunterwegs) [reply NEWS.md:24]: All #176 which was a giant PR unfortunately

**Comment by pratikunterwegs** [comment]
Thanks! Pushed fixes in https://github.com/epiverse-trace/epidemics/pull/205/commits/eb439d8d0622da59e73083673bb55718f71f68b2.

**Pull request merged**